### PR TITLE
Add buttons with register_page_header_buttons-hook

### PR DESCRIPTION
**Pull request by th3hamm0r**
see #626

### CODE REVIEW

File: wagtail_localize/wagtail_hooks.py [hunk 1]
```diff
@@ -148,8 +160,9 @@ def page_listing_more_buttons(page, page_perms, next_url=None):
         if has_locale_to_translate_to:
             url = reverse("wagtail_localize:submit_page_translation", args=[page.id])
 
-            yield wagtailadmin_widgets.Button(
-                _("Translate this page"), url, priority=60
+            yield set_button_icon(
+                wagtailadmin_widgets.Button(_("Translate this page"), url, priority=60),
+                "site",
```
Comment (th3hamm0r) [inline wagtail_localize/wagtail_hooks.py:165]: Icon ok?

File: wagtail_localize/wagtail_hooks.py [hunk 2]
```diff
@@ -159,20 +172,24 @@ def page_listing_more_buttons(page, page_perms, next_url=None):
             if next_url is not None:
                 url += "?" + urlencode({"next": next_url})
 
-            yield wagtailadmin_widgets.Button(
-                _("Sync translated pages"), url, priority=65
+            yield set_button_icon(
+                wagtailadmin_widgets.Button(
+                    _("Sync translated pages"), url, priority=65
+                ),
+                "resubmit",
```
Comment (th3hamm0r) [inline wagtail_localize/wagtail_hooks.py:179]: Icon ok?

File: wagtail_localize/wagtail_hooks.py (hunk 1 above)
Comment (th3hamm0r) [reply wagtail_localize/wagtail_hooks.py]: Wow, that was easy, but please check the svg code. I've used the "solid" version of the "language"-icon from https://heroicons.com/. It looks good to me, but I'm not a frontend-guy ;)
wagtail_localize_language_icon

Comment (zerolab) [reply wagtail_localize/wagtail_hooks.py]: The icon looks good to me locally 👍

**Pull request merged**